index.html

```diff
@@ -12410,13 +12414,25 @@ <h2>Keywords</h2>
   <p>JSON-LD <a>keywords</a> are described in <a class="sectionRef" href="#syntax-tokens-and-keywords"></a>,
     this section describes where each <a>keyword</a> may appear within different JSON-LD structures.</p>
 
+  <p>Within
+    <a>node objects</a>,
+    <a>value objects</a>,
+    <a>graph objects</a>,
+    <a>list objects</a>,
+    <a>set objects</a>, and
+    <a>nested properties</a>
+    all <a>keywords</a> other than `@context` MAY be aliased.
+    The `@context` MUST NOT bee aliased.
+    Within <a>local contexts</a> and <a>expanded term definitions</a>,
+    <a>keywords</a> MAY NOT be aliased, other than to create a <a>keyword</a> alias.</p>
```

Comment:
```suggestion
    <a>keyword</a> aliases MAY NOT used.</p>
```
I think the "other than to create a kw alias" adds confusion...